### ISSUE TITLE
Continue running

### ISSUE THREAD

**Issue by ke-s** [geekyutao/GraphIQA#2]
Excuse me. When I was training your code, it gets interrupted when the code is running half way for some reasons. Currently the model has been run to the Pretrain-model_step_1585000.pth. How should I continue training based on that？Or do I need to start training the model again? 
Thank you.

**Comment by sunsm-2020** [comment]
Is the dataset you used for pre-training Kadis700k? If so, you can check the loss. When the triplet loss is almost all 0, there is no need to continue training. Notably, triplet loss sometimes leads to unstable training. It is recommended to add type classification loss as assistance. We will update the relevant code soon.
